api/StyleMedia.json

```diff
@@ -0,0 +1,145 @@
+{
+  "api": {
+    "StyleMedia": {
+      "__compat": {
+        "support": {
+          "chrome": {
+            "version_added": "6"
+          },
+          "chrome_android": {
+            "version_added": "18"
+          },
+          "edge": {
+            "version_added": "12"
+          },
+          "firefox": {
+            "version_added": false
+          },
+          "firefox_android": {
+            "version_added": false
+          },
+          "ie": {
+            "version_added": "9"
+          },
+          "opera": {
+            "version_added": "15"
+          },
+          "opera_android": {
+            "version_added": "14"
+          },
+          "safari": {
+            "version_added": "5"
+          },
+          "safari_ios": {
+            "version_added": "5"
```

Comment:
This does not match https://github.com/mdn/browser-compat-data/blob/main/api/Window.json#L9299-L9345

Which is correct?

Reply:
Fixed!  "4" is correct (based on a quick test in BrowserStack).